wrapper/CSharp/wolfSSL_CSharp/wolfCrypt.cs

```diff
@@ -511,38 +532,11 @@ public class wolfcrypt
 #if WindowsCE
         [DllImport(wolfssl_dll)]
         private extern static IntPtr wc_GetErrorString(int error);
-        public delegate void loggingCb(int lvl, string msg);
+        /* No Windows CE decorator for logging call-back */

```

Comment:
I think this could break some users. I would prefer the logging continue to be duplicated. Technically its in wolfCrypt not wolfSSL. I don't want a refactor on this. Please duplicate the logging.

Reply:
Ok. Do you have an example of what might be problematic & why the duplication is preferred? 

I'd like to include a comment to avoid a future de-dupe.

Reply:
WolfCrypt only users who are already calling wolfcrypt.setLogging

Reply:
Hm. Interesting. 

What do you think of moving it all from wolfSSL to wolfcrypt, no duplicating, but including API references to maintain current functionality?

Reply:
That’s fine. As long as we keep the existing API’s. I do not want to break compatibility